lib/support/Day.ts

```diff
@@ -60,30 +60,50 @@ export function tz(input?: Input, timezone?: string): Day {
 /**
  * Creates a new `Ymd` object.
  */
+export function createYmd(year?: number | null, month?: number | null, date?: number | null): Ymd
+export function createYmd(day: Day): Ymd
 export function createYmd(
-  year: number | null = null,
+  yearOrDay: number | Day | null = null,
   month: number | null = null,
   date: number | null = null
 ): Ymd {
+  if (isNumber(yearOrDay) || isNullish(yearOrDay)) {
+    return {
+      year: yearOrDay,
+      month,
+      date
+    }
+  }
+
   return {
-    year,
-    month,
-    date
+    year: yearOrDay.year(),
+    month: yearOrDay.month(),
```

Comment:
I think we need to add `1` here since `day.month()` returns `0` for January. I might be wrong 👀

Reply:
updated